rocksdb.nim

```diff
@@ -192,6 +185,43 @@ proc contains*(db: RocksDBInstance, key: openArray[byte]): RocksDBResult[bool] =
   else:
     ok(false)
 
+proc del*(db: RocksDBInstance, key: openArray[byte]): RocksDBResult[bool] =
+  if key.len <= 0:
+    return err("rocksdb: key cannot be empty on del")
+
+  # This seems like a bad idea, but right now I don't want to
+  # get sidetracked by this. --Adam
+  if not db.contains(key).get:
+    return ok(false)
+
+  var errors: cstring
+  rocksdb_delete(db.db, db.writeOptions,
+                  cast[cstring](unsafeAddr key[0]), csize_t(key.len),
+                  errors.addr)
+  bailOnErrors()
+  ok(true)
+
+proc clear*(db: var RocksDBInstance): RocksDBResult[bool] =
+  # Is there a better way to do this? The best I was able to find
```

Comment:
can replace the implementation with `raiseAssert "unimplemented"` while the issue is being further discussed